rs/nns/handlers/root/impl/canister/root.did

```diff
@@ -145,16 +145,25 @@ type UpdateCanisterSettingsResponse = variant {
   Err : UpdateCanisterSettingsError;
 };
 
+type CallCanisterProposal = record {
```

Comment:
Can we rename this (and the Rust type) to CallCanisterRequest?

I think I am the one who named the Proposal, but that was when I was very new and didn't know what I was doing.